.github/workflows/quick_test.sh

```diff
@@ -9,28 +9,24 @@ set -e
 ##############################################
 # Initialize the repository
 # - Generate final version of architecture files
-# - Run FPGA tasks to validate netlist generations 
+# - Run FPGA tasks to validate netlist generations
 python3 SCRIPT/repo_setup.py --openfpga_root_path ./OpenFPGA
 
-# Post processing netlist to use custom cells  
+# Post processing netlist to use custom cells
 python3 HDL/common/custom_cell_mux_primitive_generator.py --template_netlist HDL/k4_N8_reset_softadder_caravel_io_FPGA_12x12_customhd_cc/SRC/sub_module/mux_primitives.v --output_verilog HDL/k4_N8_reset_softadder_caravel_io_FPGA_12x12_customhd_cc/SRC/sub_module/mux_primitives_hd.v
 
 ##############################################
 # Generate wrapper HDL codes to bridge Caravel I/Os and FPGA I/Os
 python3 HDL/common/wrapper_lines_generator.py --template_netlist HDL/common/caravel_fpga_wrapper_hd_template.v --pin_assignment_file HDL/common/caravel_wrapper_pin_assignment_v1.0.json --output_verilog HDL/common/caravel_fpga_wrapper_hd_v1.0.v
 python3 HDL/common/wrapper_lines_generator.py --template_netlist HDL/common/caravel_fpga_wrapper_hd_template.v --pin_assignment_file HDL/common/caravel_wrapper_pin_assignment_v1.1.json --output_verilog HDL/common/caravel_fpga_wrapper_hd_v1.1.v
 
+##############################################
+# Generate wrapper HDL codes to bridge Caravel I/Os and FPGA I/Os
```

Comment:
It seems that there are some conflicts on this file, please check out the latest version.

Reply:
I should reset this file, I merged with old master.